website/docs/plugin/sdkv2/testing/acceptance-tests/testcase.mdx

```diff
@@ -95,7 +95,7 @@ currently running.
 
 `TestCase` offers several fields for developers to add to customize and validate
 each test, defined below. The source for `TestCase` can be viewed [here on
-godoc.org](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCase)
+godoc.org](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCase)
```

Comment:
```suggestion
godoc.org](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/helper/resource#TestCase)
```

Perhaps this should reference `terraform-plugin-testing` now that Terraform Provider testing has migrated to a new [terraform-plugin-testing](https://developer.hashicorp.com/terraform/plugin/testing) Go module?